src/SchematicHQ.Client.Test/Webhooks/WebhookVerifierTests.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using SchematicHQ.Client.Webhooks.WebhookUtils;
+
+namespace SchematicHQ.Client.Test.Webhooks
+{
+    [TestFixture]
+    public class WebhookVerifierTests
+    {
+        private const string WebhookSecret = "test_secret";
+        private const string RequestBody = "{\"event\":\"test_event\"}";
+        private const string Timestamp = "1234567890";
+
+        private string _validSignature;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _validSignature = WebhookVerifier.ComputeHexSignature(RequestBody, Timestamp, WebhookSecret);
+        }
+
+        [Test]
```

Comment:
Maybe add a test that tries to verify with the _wrong_ secret `"wrong_secret"`

Reply:
Thanks, done